modules/ROOT/partials/configuration/iframe_template_callback.adoc

```diff
@@ -0,0 +1,24 @@
+[[iframe_template_callback]]
+== `+iframe_template_callback+`
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+This option allows you to specify the function that will return the HTML embed code of the iframe media that you are attempting to insert into {productname}.
```

Comment:
This doesn't follow our style guide as we shouldn't be using pronouns such as `you`: https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#pronouns. I'm guessing it's like this for all the other `*_template_callback` options though since they are quite old?